### ISSUE TITLE
The text overflows in the cards of PACS

### ISSUE THREAD

**Issue by AryaArunB** [FNNDSC/ChRIS_ui#390]
<img width="1440" alt="Screenshot 2022-04-09 at 10 34 10 AM" src="https://user-images.githubusercontent.com/60646682/162557209-71b7e1c8-c3a7-47ae-b3f7-b1a90680bb5b.png">
This issue is open feel free to contribute!

**Comment by archijaiswal** [comment]
Thanks @AryaArunB, I'm on it!

**Comment by aparna-bhatt** [comment]
Hey @archijaiswal are you on it?

**Comment by archijaiswal** [comment]
Yes @aparna-bhatt, feel free to share your suggestion if you've any. Thanks!

**Comment by ritikalohia** [comment]
@aparna-bhatt , Is anyone working on this issue or not?

**Comment by aparna-bhatt** [comment]
@archijaiswal are you working on this issue?

**Comment by archijaiswal** [comment]
Actually, I'm having hard time recreating this bug. In PACS lookup system, the search result is taking a long time and ending up with no result found when I'm searching for the patient "ANON HANNAH CARTER". Will you mind recreating this bug or sharing any result found in PACS Lookup system? Thanks

**Comment by AryaArunB** [comment]
just try entering the patient id 7654321

**Comment by aparna-bhatt** [comment]
@archijaiswal yeah I am also facing the same issue. @AryaArunB  I also tried the patient id 7654321 but still it is taking long to search and ending up with no results.

**Comment by aparna-bhatt** [comment]
@jennydaman can you help us by providing solution to this problem .

**Comment by jennydaman** [comment]
Heey @aparna-bhatt we're aware of the problem and unfortunately the server broke in an unexpected way, it's going to stay down for a bit.

**Comment by aparna-bhatt** [comment]
@jennydaman ok .
